bindings/python/docs/source/supported_types.rst

```diff
@@ -0,0 +1,34 @@
+.. _type support:
+
+Supported Types
+===============
+
+PyMongoArrow currently supports a small subset of all BSON types.
+Support for additional types will be added in subsequent releases.
+
+.. note:: For more information about BSON types, see the
+   `BSON specification <http://bsonspec.org/spec.html>`_.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - BSON Type
+     - Type Identifiers
```

Comment:
In the Schema docstring it mentions that bson type are also supported. Should we document them in this table?

Reply:
Ah, right forgot about those.